resources/css/ceres.scss

```diff
@@ -10679,6 +10679,18 @@ a:active {
 .border-radius-0 {
     border-radius: 0;
 }
+
+@include media-breakpoint-down(xs){
+    .only-button-mobile {

```

Comment:
Die Benennung der CSS-Klasse ist vielleicht missverständlich. Meiner Meinung nach wird hier eher der Eindruck vermittelt, dass nur auf mobile der Button angezeigt wird. Außerdem würde ich aus Gründen der Einheitlichkeit nicht `-mobile` sondern `-xs` verwenden. Man könnte dann auch darüber nachdenken, diese Klassen auch für andere Breakpoints bereitzustellen.
In der Summe vielleicht dann so:
```
.btn-only-icon-xs {}
.btn-only-icon-sm {}
.btn-only-icon-md {}
.btn-only-icon-lg {}
```

Hier ein Beispiel, wie man das ganze generisch bauen kann:
[`/resources/scss/bootstrap/scss/utilities/_visibility.scss`](https://github.com/plentymarkets/plugin-ceres/blob/stable/resources/scss/bootstrap/scss/utilities/_visibility.scss#L13)

Reply:
Finde ich eine sehr gute Idee! Werde das ganze dementsprechend anpassen.